stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -69,6 +89,25 @@ private static String readContentsAsUTF8(InputStream in) throws IOException {
     return out.toString("UTF-8");
   }
 
+  private String prettyPrintContentWithTimeOut(
+      AsyncPrettyPrinter asyncPrettyPrinter,
+      InputStream in) throws IOException {
+    AsyncPrettyPrintingTask prettyPrintingTask = new AsyncPrettyPrintingTask(
+        in,
+        asyncPrettyPrinter);
+    ExecutorService executor = Executors.newSingleThreadExecutor();
+    Future<String> future = executor.submit(prettyPrintingTask);
+    String prettifiedContent = "";
```

Comment:
This isn't needed and is also considered an anti-pattern.  Functions that return magic, undocumented values can be extremely frustrating to debug in large programs.  Avoid this whenever possible and instead use exceptions (even crashing the app can be better than this unpredictable behaviour).

For instance, imagine if you were a user and you were seeing empty payloads for some requests but not others and there were no error messages related to the problem.  What a frustrating problem to encounter...